### PR TITLE
update doc to describe exclude feature

### PR DESCRIPTION
**Pull request by hkbarton**
## What has been implemented?

Updated documentation to describe the exclude feature.

## Steps to verify

N/A

## Todos:

N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,36 +38,54 @@ inputs: # The configuration the Component accepts according to its docs
 
 # Documentation
 
+- [Documentation](#documentation)
```
Comment (eahefnawy) [inline README.md:41]: this one is really not necessary imo

File: README.md [hunk 2]
```diff
@@ -38,36 +38,54 @@ inputs: # The configuration the Component accepts according to its docs
...
 - [Quick-Start](#quick-start)
 - [Features](#features)
+    - [Simplicity](#simplicity)
+    - [Instant Deployments](#instant-deployments)
+    - [Build Your Own](#build-your-own)
+    - [Registry](#registry)
+    - [Serverless](#serverless)
```
Comment (eahefnawy) [inline README.md:48]: These are also not important to list in the ToC imo

File: README.md [hunk 3]
```diff
@@ -38,36 +38,54 @@ inputs: # The configuration the Component accepts according to its docs
...
 - [CLI Commands](#cli-commands)
-- [Templates](./templates)
-- [Artwork](https://github.com/serverless/artwork)
-- [Join Us on Slack](https://serverless.com/slack)
+      - [`serverless registry`](#serverless-registry)
+      - [`serverless registry publish`](#serverless-registry-publish)
+      - [`serverless deploy`](#serverless-deploy)
+      - [`serverless remove`](#serverless-remove)
+      - [`serverless info`](#serverless-info)
+      - [`serverless dev`](#serverless-dev)
```
Comment (eahefnawy) [inline README.md:88]: This list does not render properly
https://github.com/serverless/components/blob/50003a2305d45cddc6f424ee949cfc3fa4ee7697/README.md

File: README.md [hunk 4]
```diff
@@ -38,36 +38,54 @@ inputs: # The configuration the Component accepts according to its docs
...
+      - [AWS Credentials](#aws-credentials)
+      - [Google Credentials](#google-credentials)
+    - [Stages](#stages)
+    - [Variables](#variables)
+      - [Variables: Org](#variables-org)
+      - [Variables: Stage](#variables-stage)
+      - [Variables: App](#variables-app)
+      - [Variables: Name](#variables-name)
+      - [Variables: Environment Variables](#variables-environment-variables)
+      - [Variables: Outputs](#variables-outputs)
```
Comment (eahefnawy) [inline README.md:67]: This is also an overkill

File: README.md [hunk 5]
```diff
@@ -38,36 +38,54 @@ inputs: # The configuration the Component accepts according to its docs
...
-  - [Variables](#variables)
+    - [serverless.yml](#serverlessyml)
+    - [Inputs](#inputs)
+    - [Deploying](#deploying)
+    - [State](#state)
+    - [Versioning](#versioning)
+    - [Outputs](#outputs)
+    - [Credentials](#credentials)
+      - [AWS Credentials](#aws-credentials)
+      - [Google Credentials](#google-credentials)
```
Comment (eahefnawy) [inline README.md:59]: Ok I see the value in listing supported credentials, and it doesn't make the list too long. So I like this one

File: README.md [hunk 6]
```diff
@@ -38,36 +38,54 @@ inputs: # The configuration the Component accepts according to its docs
...
+    - [Adding The Serverless Agent](#adding-the-serverless-agent)
+    - [Development Workflow](#development-workflow)
+    - [Development Tips](#development-tips)
+      - [Start With The Outcome](#start-with-the-outcome)
+      - [Knowing The Outcome Is An Advantage](#knowing-the-outcome-is-an-advantage)
+      - [Keep Most State On The Cloud Provider](#keep-most-state-on-the-cloud-provider)
+      - [Store State Immediately After A Successful Operation](#store-state-immediately-after-a-successful-operation)
+      - [Optimize For Accessibility](#optimize-for-accessibility)
+      - [No Surprise Removals](#no-surprise-removals)
+      - [Write Integration Tests](#write-integration-tests)
```
Comment (eahefnawy) [inline README.md:81]: No need to list this one either

File: README.md [hunk 7]
```diff
@@ -267,6 +285,18 @@ inputs:
...
+```yaml
+inputs:
+  src:
+    src: ./src # Source files
+    exclude:
+      - .env # exclude .env file in ./src
+      - '.git/**' # exclude .git folder and all subfolders and files inside it
+      - '**/*.log' # exclude all files with .log extension in any folder under the ./src
+```
+
```
Comment (eahefnawy) [inline README.md:299]: Looks great!

**Comment by hkbarton** [comment]
hmm, I think my VSCode automatically updates those TOC for me. let's keep this PR simple to just add doc for exclude, and revert those TOC update. @eahefnawy

**Comment by hkbarton** [comment]
@eahefnawy updated PR and kept the one you like 😉

**Comment by hkbarton** [comment]
updated Chinese version as well.

**Comment by hkbarton** [comment]
@eahefnawy updated

**Pull request merged**